### PR TITLE
tests: Add tests webpack dev server

### PR DESCRIPTION
**Pull request by lmiller1990**
Add some basic tests for npm/webpack-dev-server.

### CODE REVIEW

File: npm/webpack-dev-server/package.json [hunk 1]
```diff
@@ -16,6 +17,10 @@
   },
   "devDependencies": {
     "@types/webpack-dev-server": "^3.11.1",
+    "chai": "^4.2.0",
+    "mocha": "^8.1.3",
```
Comment (elevatebart) [inline npm/webpack-dev-server/package.json:21]: do we need both mocha and TS mocha ? 🤔

File: circle.yml [hunk 2]
```diff
@@ -908,6 +908,16 @@ jobs:
           working_directory: npm/webpack-preprocessor/examples/react-app
       - store-npm-logs
 
+  npm-webpack-dev-server:
```
Comment (elevatebart) [inline circle.yml:911]: I believe this stage need to be mentioned in the `jobs` section in order to be run on CI.

**Review by elevatebart — APPROVED** [review]
Smallest change ever might be done later.

File: npm/webpack-dev-server/package.json (hunk 1 above)
Comment (lmiller1990) [reply npm/webpack-dev-server/package.json:21]: Yep, ts-mocha has mocha as a peer dep - better to lock the version.

File: circle.yml (hunk 2 above)
Comment (lmiller1990) [reply circle.yml:911]: done!

**Pull request merged**